networkx/algorithms/shortest_paths/generic.py

```diff
@@ -444,7 +444,7 @@ def path_length(v):
             all_pairs = nx.floyd_warshall(G, weight=weight)
             s = sum(sum(t.values()) for t in all_pairs.values())
         elif method == "floyd-warshall-numpy":
-            s = nx.floyd_warshall_numpy(G, weight=weight).sum()
+            s = nx.floyd_warshall_numpy(G, weight=weight).sum().tolist()
```

Comment:
The expression in the return statement, `return s / (n * (n - 1))`, only operates on scalars, so it's not vectorized. However, after taking a closer look at this, it appears that `s` is always a float, so I'll use `float(...)` instead of `.tolist()`.